lib/beaker/dsl/install_utils.rb

```diff
@@ -1279,6 +1284,10 @@ def install_puppetagent_dev_repo( host, opts )
         when /^(debian|ubuntu|cumulus)$/
           on host, "dpkg -i --force-all #{onhost_copied_file}"
           on host, "apt-get update"
+        when /^windows$/
+          result = on host, "echo #{onhost_copied_file}"
+          onhost_copied_file = result.raw_output.chomp
+          on host, "cmd /C 'start /w #{onhost_copied_file}'"
```

Comment:
If you use {:cmdexe => true} in a command object it will correctly append 'cmd /C' to the command string.